README.md

```diff
@@ -1,25 +1,112 @@
 # Watch (me nae-nae) bot
 
-## Backend
-Our server implementation basically has two things: a polling function that updates the database with face encodings whenever there are new entries, and a socket handler that will handle all the communication between a client and the Pi.
-
-There are two processes: the main process and the poll process:
-- The poll process will synchronize the shared memory dictionary of student IDs aand encodings, and also update the SQL database with the encodings whenever necessary
-- The main process will run two threads: face recognition and the socket interface; the face recognition thread will process the frame every second, and the socket interface will maintain the event loop to handle incoming socket connections (two from the Pi for camera and parameters, and two from a client for the camera and parameters). The flow of data from elements are as:
-```
-        |------------|<-- params --|----------|<-- params --|------------|
-        |     Pi     |             |    VM    |             |   Client   |
-        |------------|-- frames -->|----------|-- frames -->|------------|
-```
-
-## Website
-Our website has a homepage (of a random design), and a student and teacher area.
-
-### Student
-The student area will allow students to put in their student ID, name, photo, and a secret (provided by the teacher) to register their face into the database.
-
-### Teacher
-The teacher area will allow the teacher to:
+Watch Bot is an automatic attendance taking solution to simplify keeping track of student presence in classroom and laboratory environments. It uses facial recognition to analyze images received from a camera module mounted on a robot controlled by a Raspberry Pi to update a remote database when it detects the presence of a registered student. All of the information on the database can be viewed and controlled by the instructor of the class using a web application, including a live feed of what the robot is capturing.
+
+**Table of contents:**
+
+- [Technologies/Concepts](#technologies/concepts)
+- [Project Components](#project-components)
+- [Overall Flow of Information](#overall-information-flow)
+<!-- - Contributions section perhaps? -->
+
+## Technologies/Concepts
+
+### Technologies
+
+- Raspberry Pi
+- <a href="https://github.com/ageitgey/face_recognition" target="_blank"><code>face_recognition</code> Python library</a>
```

Comment:
It might be better to use markdown link formatting instead.

Reply:
It's just easier to do the new tab opening stuff. I can fix it still if you prefer it that way

Reply:
You mean to be able to click on images? 

You can chain markdown:

[![alt_text](https://i.imgur.com/GGTkAvq.gif)](https://i.imgur.com/GGTkAvq.gif)


`[![alt_text](image link)](whatever link)`